.github/workflows/release.yaml

```
@@ -19,7 +19,7 @@ jobs:
 
       - name: Set up dependencies
         run: |
-          python3.8 -m venv .venv
+          python3.9 -m venv .venv
```

Consider 
```suggestion
          python3 -m venv .venv
```
to make it less problematic next time we need to bump the minimum version of Python

---

ok fixed